packages/wrangler/src/dev/miniflare.ts

```diff
@@ -353,10 +360,26 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 					}
 				})
 				.join("\n"),
-	};
+	});
 
 	if (bindings.ai?.binding) {
-		config.serviceBindings[bindings.ai.binding] = AIFetcher;
+		externalWorkers.push({
+			name: EXTERNAL_AI_WORKER_NAME,
+		  modules: [
+				{ type: "ESModule", path: "index.mjs", contents: EXTERNAL_AI_WORKER_SCRIPT }
+			  ],
+			serviceBindings: {
+				FETCHER: AIFetcher,
```

Comment:
Will this conflict if users have their own binding called `FETCHER`? Perhaps naming it something more obscure like `__WORKERS_INTERNAL_AI_FETCHER` might help prevent this?